cmd/lakectl/cmd/abuse_commit.go

```diff
@@ -48,10 +48,14 @@ var abuseCommitCmd = &cobra.Command{
 		generator.Run(func(input chan string, output chan stress.Result) {
 			ctx := cmd.Context()
 			client := getClient()
+			allowEmpty := true
 			for work := range input {
 				start := time.Now()
 				resp, err := client.CommitWithResponse(ctx, u.Repository, u.Ref, &apigen.CommitParams{},
-					apigen.CommitJSONRequestBody(apigen.CommitCreation{Message: work}))
+					apigen.CommitJSONRequestBody(apigen.CommitCreation{
+						Message:    work,
+						AllowEmpty: &allowEmpty,
```

Comment:
Why do we now need this?

Reply:
Because we blocked empty commits without explicit flag a long time ago and didn't modify the abuse command - the command was broken!